app/models/traderDetails/representative/RepresentativeCapacity.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.traderDetails.representative
+
+import models.{RadioModel, WithName}
+
+sealed trait RepresentativeCapacity
+
+object RepresentativeCapacity extends RadioModel[RepresentativeCapacity] {
+
+  case object Option1 extends WithName("option1") with RepresentativeCapacity
```

Comment:
I'm not sure what has been done on other enumerables so far but can these be modelled as Direct and Indirect. I think however on the latest revision of the journey there is only Indirect as an 'option' therefore we would not ask it. It's one worth clarifying with Danny as it is meant to be driven by reference data.